contracts/truefi2/LoanFactory2.sol

```diff
@@ -33,6 +35,9 @@ contract LoanFactory2 is ILoanFactory2, Initializable {
     // @dev Track Valid LoanTokens
     mapping(address => bool) public override isLoanToken;
 
+    // @dev Track valid debtTokens
+    mapping(address => bool) public isDebtToken;
```

Comment:
Minor refactor (task added to bottom of backlog): Let's map from DebtToken to bool to save unnecessary casts elsewhere.